### PR TITLE
Added Documentation for Logger variables in README

### PR DESCRIPTION
**Pull request by jwuensche**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,35 @@ At the moment, no live-reload of certificates is supported, so to use new certif
 The TLS endpoint also supports TLS client authentication, which can be used to create sophisticated authentication and authorization structures easily.
 The intention is to have the TLS endpoint exposed to the inner application (your **backend**) and to have the WS endpoint exposed to your end users via a reverse proxy like nginx (which also serves the files for your **frontend**) which further separates TLS termination away from `autobahnkreuz` to your cloud provider/environment. The separation between TLS for internal usage and WS for external usage also yields a better privilege separation and therefore improves application security.
 
+## Logging Configuration
```
Comment (johannwagner) [inline README.md]: Header is too big. Should be `###`.

File: README.md [hunk 2]
```diff
@@ -65,6 +65,35 @@ At the moment, no live-reload of certificates is supported, so to use new certif
...
+`SERVICE_LOGLEVEL`
+
+This allows setting the depth which the Logger will display at maximum. Values are defined as: 
+
+`CRITICAL`  
+`ERROR`  
+`WARN`  
+`INFO`  
+`DEBUG`  
+
```
Comment (johannwagner) [inline README.md]: Maybe you want to introduce a list here.

File: README.md [hunk 3]
```diff
@@ -65,6 +65,35 @@ At the moment, no live-reload of certificates is supported, so to use new certif
 The TLS endpoint also supports TLS client authentication, which can be used to create sophisticated authentication and authorization structures easily.
 The intention is to have the TLS endpoint exposed to the inner application (your **backend**) and to have the WS endpoint exposed to your end users via a reverse proxy like nginx (which also serves the files for your **frontend**) which further separates TLS termination away from `autobahnkreuz` to your cloud provider/environment. The separation between TLS for internal usage and WS for external usage also yields a better privilege separation and therefore improves application security.
 
+## Logging Configuration
+Configuration of the `autobahnkreuz` Logger is supported by setting following environment variables:
+
+`SERVICE_LOGFORMAT`
```
Comment (johannwagner) [inline README.md]: You also want to structure this better.

Comment (jwuensche) [reply README.md]: Changed var names to subsections to be navigable in section trees.

File: README.md (hunk 2 above)
Comment (jwuensche) [reply README.md]: Included that in new version

File: README.md (hunk 1 above)
Comment (jwuensche) [reply README.md]: Fixed

**Pull request merged**